src/client/lib/decorators/socket.js

```diff
@@ -12,16 +12,16 @@ export function communicator(target) {
     }
   });
   Object.defineProperty(target.prototype, 'socket', {
-    set: function(socket) {
+    set(socket) {
       this._socket = socket;
       this._initializeSocketComm(socket);
     },
-    get: function() {
+    get() {
       return this._socket;
     }
   });
   Object.defineProperty(target.prototype, 'detached', {
-    value: function() {
+    value() {
```

Comment:
'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).